src/runtime_src/core/common/ishim.h

```diff
@@ -391,38 +391,39 @@ struct shim : public DeviceType
   }
 
   void
-  copy_bo(xclBufferHandle dst, xclBufferHandle src, size_t size, size_t dst_offset, size_t src_offset) override
+  copy_bo(xrt_buffer_handle dst, xrt_buffer_handle src, size_t size, size_t dst_offset, size_t src_offset) override
   {
-    if (auto err = xclCopyBO(DeviceType::get_device_handle(), dst, src, size, dst_offset, src_offset))
+    if (auto err = xclCopyBO(DeviceType::get_device_handle(),
+      to_xclBufferHandle(dst), to_xclBufferHandle(src), size, dst_offset, src_offset))
       throw system_error(err, "unable to copy BO");
   }
 
   void
-  sync_bo(xclBufferHandle bo, xclBOSyncDirection dir, size_t size, size_t offset) override
+  sync_bo(xrt_buffer_handle bo, xclBOSyncDirection dir, size_t size, size_t offset) override
   {
-    if (auto err = xclSyncBO(DeviceType::get_device_handle(), bo, dir, size, offset))
+    if (auto err = xclSyncBO(DeviceType::get_device_handle(), to_xclBufferHandle(bo), dir, size, offset))
```

Comment:
See earlier comment regarding reformatting the line.

Reply:
Broke into two lines.